hugo/content/exporter/_index.md

```diff
@@ -312,3 +323,36 @@ If you need to run multiple postgres_exporter services, follow the same instruct
     - Update the Prometheus auto.d target file to include the new exporter in the same one you already had running for this system
 
 Remaining steps to initialize service at boot and start it up should be the same as above for the default service.
+
+### Win Server 2012R2
+
+Currently the Windows installers assume you are logged in the local Administrator account, so please ensure to do so before attempting the following.
+
+Install the WMI and PostgreSQL exporters by:
+
+1. Find and double-click the `pgMonitor_client_1.0_Crunchy.win.x86_64.exe` file previously downloaded from the [Crunchy Customer Portal](https://access.crunchydata.com/). It will present you with the following screen:
```

Comment:
Not available on portal

Reply:
It won't ever be? Then where will random Win customers get it?

Reply:
Don't know.

Reply:
fixed